vspace/src/main/java/edu/asu/diging/vspace/web/staff/UpdateContentOrderController.java

```diff
@@ -23,17 +30,24 @@ public class UpdateContentOrderController {
 
     @Autowired
     private IContentBlockManager contentBlockManager;
+    
+    @Autowired
+    private ISlideManager slideManager;
 
     @RequestMapping(value = "/staff/module/{moduleId}/slide/{id}/blocks/order/update", method = RequestMethod.POST)
-    public ResponseEntity<List<ContentBlock>> adjustContentOrder(@RequestBody List<ContentBlock> contentBlockList) {
+    public ResponseEntity<List<ContentBlock>> adjustContentOrder(@RequestBody String contentBlockString, @PathVariable("id") String slideId) {
 
+        ObjectMapper objectMapper = new ObjectMapper();
+        ISlide slide = slideManager.getSlide(slideId);
+        List<ContentBlock> contentBlockList;
         try {
-            contentBlockManager.updateContentOrder(contentBlockList);
-        } catch (BlockDoesNotExistException e) {
-            logger.warn("Block Id does not exist, bad request.", e);
-            return new ResponseEntity<List<ContentBlock>>(HttpStatus.BAD_REQUEST);
+            System.out.println(contentBlockString);
```

Comment:
?

Reply:
i was trying to fix the content ordering, but realised that it is not a part of this ticket so I left that part

Reply:
so why is there a system.out.println in there?

Reply:
That was for checking the contents of the JSON stringified object, removed the print statement now.